data/grub_auth/grub_passwd.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+GRUB_PASSWD=$1
+
+grub2-mkpasswd-pbkdf2 <<EOF
+$GRUB_PASSWD
+$GRUB_PASSWD
+EOF
```

Comment:
This shell script is not very robust as there is no "expect" usage. But since it works revising it or not is up to you.

Reply:
Yup, thanks, then keep it